index.html

```diff
@@ -19,6 +19,14 @@ <h5 class="header col s12 light">
           Who are you?
           </a>
       </div>
+      <div class="row center">
+          <br /><br /><br />
```

Comment:
No idea. I just saw that <br />s were used as spacers up above and I didn't really want to dive into the CSS.